java/apps/authentication-portal/src/main/webapp/add-security-questions.jsp

```diff
@@ -97,18 +93,16 @@
 </head>
 <body class="login-portal layout authentication-portal-layout">
     <layout:main layoutName="<%= layout %>" layoutFileRelativePath="<%= layoutFileRelativePath %>" data="<%= layoutData %>" >
-        <layout:component componentName="ProductHeader">
+        <layout:component componentName="ProductHeader" >
```

Comment:
Remove whitespace. Check in other places as well

Reply:
Fixed with e89ab17